tests/test_repair.py

```diff
@@ -468,11 +506,13 @@ def test_legacy_compare_with_legacy_result2():
     for sec in obj._find_intact_sub_trees():
         intacts[obj.repair_type_map[sec]].append(sec)
 
-    assert_equal([sec.id + offset for sec in intacts[RepairType.trunk]],
-                 [])
-    assert_equal([sec.id + offset for sec in intacts[RepairType.oblique]],
-                 [])
-    assert_equal({sec.id + offset for sec in intacts[RepairType.tuft]},
-                 set())
+    # Since there is no apical dendrite, all of those are empty
+    for extended_type in [RepairType.trunk, RepairType.oblique, RepairType.tuft]:
+        assert_equal(intacts[RepairType.trunk], [])
```

Comment:
should be extended_type not RepairType.trunk and I proposed the length because RepairType.tuft is a set() so this will fail

Reply:
Ah yes, sorry for that.
No, `intact[extended_type]` is not a set but a list. The set were previously created afterwards with a set comprehension in order to make the diff more easily readable when there was a mismatch.
But here since we expect an empty list, they are not needed.